docs/userguide/icu4c-readme.md

```diff
@@ -60,25 +57,25 @@ The ICU homepage also links to related information about writing internationaliz
 
 **Important:** Please make sure you understand the [Copyright and License Information](http://source.icu-project.org/repos/icu/trunk/icu4c/LICENSE).
 
-## What Is New In This Release?
+## What Is New In The Current Release?
 
-> :construction: TODO: move this section elsewhere
+See the [ICU download page](http://site.icu-project.org/download/) to find the subpage for the current release, including any other changes, bug fixes, known issues, changes to supported platforms and build environments, and migration issues for existing applications migrating from previous ICU releases.
 
-See the [ICU 67 download page](http://site.icu-project.org/download/67) for more information on this release, including any other changes, bug fixes, known issues, changes to supported platforms and build environments, and migration issues for existing applications migrating from previous ICU releases.
+The subpage for the current release will also include an API Change Report, both for ICU4C and ICU4J, for a complete list of APIs added, removed, or changed in this release.
 
-See the [API Change Report](APIChangeReport.html) for a complete list of APIs added, removed, or changed in this release.
+The list of API changes since the previous ICU4J release is available [here](https://htmlpreview.github.io/?https://raw.githubusercontent.com/unicode-org/icu/master/icu4c/APIChangeReport.html).
 
-<a name="RecentPreviousChanges" id="RecentPreviousChanges"></a>For changes in previous releases, see the main [ICU download page](http://site.icu-project.org/download) with its version-specific subpages.
+Changes in previous releases can also be found the main [ICU download page](http://site.icu-project.org/download) in its version-specific subpages.
 
 ## How To Download the Source Code
 
 There are two ways to download ICU releases:
 
-*   **Official Release Snapshot:**  
-    If you want to use ICU (as opposed to developing it), you should download an official packaged version of the ICU source code. These versions are tested more thoroughly than day-to-day development builds of the system, and they are packaged in zip and tar files for convenient download. These packaged files can be found at [http://site.icu-project.org/download](http://site.icu-project.org/download).  
-    The packaged snapshots are named `icu-nnnn.zip` or `icu-nnnn.tgz`, where nnnn is the version number. The .zip file is used for Windows platforms, while the .tgz file is preferred on most other platforms.  
+*   **Official Release Snapshot:**
+    If you want to use ICU (as opposed to developing it), you should download an official packaged version of the ICU source code. These versions are tested more thoroughly than day-to-day development builds of the system, and they are packaged in zip and tar files for convenient download. These packaged files can be found at [http://site.icu-project.org/download](http://site.icu-project.org/download).
```

Comment:
I wonder if it might be worth adding a brief note here that there are occasionally additional fixes put into the "`maint`" branches (?).
On the ICU download page there is already this note:
> Note: There may be additional commits on the maint-68 branch that are not included in the prepackaged download files.

Maybe something like:
> Note: There may be additional commits on the "maint" branch for a particular version that are not included in the prepackaged download files.

Or perhaps just leave this for now as-is since this PR is mostly for formatting changes (?).

Reply:
Done. That sounds like a good thing to add, so I added it.